### ISSUE TITLE
Pathname on backward button / handwritten link not giving the current path, but the path we're navigating to

### ISSUE THREAD

**Issue by tang1304** [RRebois/ft_transcendence#41]

**Comment by RhesusP** [comment]
I admit that I don't understand what is the expected result... Please, could you provide some examples to help me to understand your expectations ?

**Comment by RhesusP** [comment]
Considering we have the following code at the beginning of [navigate function](https://github.com/RRebois/ft_transcendence/blob/5cd44ad07fb99baeb169a4b9b7bd6effdb919aaf/frontend/app/js/spa-router/Router.js#L34C2-L34C42) : 
```js
async navigate(path, pushState = true) {
   console.log('destination path: ', path);
   console.log('current path: ', window.location.pathname);
``` 

Issues are the following : 
- [ ] When clicking on "go back", current path and destination path are destination path
- [ ] Same when refreshing

**Comment by RhesusP** [comment]
Closed with commit [83515dc](https://github.com/RRebois/ft_transcendence/commit/83515dc5a550abb73dcca0a82e40a6873402a9b7)
